modules/clickhouse/src/test/java/org/testcontainers/junit/clickhouse/SimpleClickhouseTest.java

```diff
@@ -1,26 +1,44 @@
 package org.testcontainers.junit.clickhouse;
 
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.mysqlclient.MySQLConnectOptions;
+import io.vertx.mysqlclient.MySQLConnection;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
 import org.junit.Test;
+import org.testcontainers.containers.ClickHouseInit;
 import org.testcontainers.containers.ClickHouseContainer;
-import org.testcontainers.db.AbstractContainerDatabaseTest;
 
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import java.util.concurrent.CountDownLatch;
 
-import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
-import static org.testcontainers.ClickhouseTestImages.CLICKHOUSE_IMAGE;
+import static org.junit.Assert.assertFalse;
 
-public class SimpleClickhouseTest extends AbstractContainerDatabaseTest {
+public class SimpleClickhouseTest {
 
     @Test
-    public void testSimple() throws SQLException {
-        try (ClickHouseContainer clickhouse = new ClickHouseContainer(CLICKHOUSE_IMAGE)) {
-            clickhouse.start();
+    public void testRawMysql() throws Throwable {
+        ClickHouseContainer clickHouseContainer = new ClickHouseContainer();
+        clickHouseContainer.start();
 
-            ResultSet resultSet = performQuery(clickhouse, "SELECT 1");
+        MySQLConnectOptions connectOptions = new MySQLConnectOptions()
+            .setPort(clickHouseContainer.getMappedPort(ClickHouseInit.MYSQL_PORT))
```

Comment:
I feel like having the constant in `ClickHousInit` makes it a bit harder to discover for users.

Reply:
The constants are shared between all three.

What do you think about adding this to ClickHouseContainer?

```
    public Integer getHttpPort() {
        return getMappedPort(ClickHouseInit.HTTP_PORT);
    }

    public Integer getNativePort() {
        return getMappedPort(ClickHouseInit.NATIVE_PORT);
    }

    public Integer getMysqlPort() {
        return getMappedPort(ClickHouseInit.MYSQL_PORT);
    } 

```